app/org/sagebionetworks/bridge/dynamodb/DynamoInitializer.java

```diff
@@ -315,7 +311,9 @@ static void initTables(final List<TableDescription> tables) {
                 logger.info("Creating table " + table.getTableName());
                 DYNAMO.createTable(createTableRequest);
             } else {
-                compareSchema(table, existingTables.get(table.getTableName()));
+                final TableDescription existingTable = existingTables.get(table.getTableName());
+                compareSchema(table, existingTable);
+                updateThroughput(table,existingTable);
```

Comment:
I agree with these remarks. We don't want to limit our ability to scale horizontally by requiring we change and deploy code to do it.